core/src/main/java/io/cheesecakelabs/core/utils/extensions/EditText+Maks.kt

```diff
@@ -0,0 +1,67 @@
+package io.cheesecakelabs.core.utils.extensions
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.EditText
+
+fun EditText.maskCPF() {
+    mask("###.###.###-##")
+}
+
+fun EditText.maskCEP() {
+    mask("#####-###")
+}
+
+fun EditText.maskCreditCard() {
+    mask("#### #### #### ####")
+}
+
+fun EditText.maskCellPhone(hasCountryCode: Boolean) {
+    if (hasCountryCode) mask("+## (##) #########")
+    else mask("(##) #########")
+}
+
+fun EditText.mask(pattern: String) {
+    this.addTextChangedListener(object : TextWatcher {
+        var isUpdating: Boolean = false
+        var previousString: String = ""
+
+        override fun afterTextChanged(s: Editable?) {}
+
+        override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {}
+
+        override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
+            val str = s.toString().replace("[./() \\-+]".toRegex(), "")
```

Comment:
I don't know if it's a good idea to do something so generic here. Maybe some masks would need one of this characters. One option is to have different TextWatchers for each mask. I remember seeing this on Nobis project: 
https://github.com/CheesecakeLabs/local-android/tree/dev/app/src/main/java/com/nobis/app/utils/text_watchers